.github/ISSUE_TEMPLATE/product-type-listing-request.md

```diff
@@ -50,8 +50,3 @@ assignees: ''
 
 - [ ] Craft transaction data payload
 - [ ] AB member to update onchain data
-- [ ] Update [SDK](https://github.com/NexusMutual/sdk)
-- [ ] Update [frontend-next](https://github.com/NexusMutual/frontend-next)
-- [ ] Update [frontend-react](https://github.com/NexusMutual/frontend-react)
-- [ ] Update [notification-processor](https://github.com/NexusMutual/notification-processor)
-- [ ] Update [event-scanner](https://github.com/NexusMutual/event-scanner) (if there are new private products)
```

Comment:
Was this intended? 🤔

Reply:
I think this was my commit. I should have probably left SDK/FE and only removed BE services tasks.

BE services doesnt need to be updated on new product types only on new product listing

Reply:
added back SDK/FE tasks on the ticklist